data/human/culture conversations.txt

```diff
@@ -752,3 +752,42 @@ mission "Burthen Triathlon"
 			label "end"
 			`	Abernathy doesn't slow down until he's within an arm's length of the finishing line; as he crosses it he seizes the tape, making a yelp of simultaneous joy and relief. He holds the tape up before everybody in the crowd and at the cameras pointed at him, taking in the attention (and catching his breath) before accepting a bottle of water from someone on the sidelines.`
 				decline
+
+mission "CCOR Guerrilla Standoff"
+	minor
+	source "Gagarin"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Although every map of human space places Gagarin in the list of seedy, go-at-your-own-risk pirate ports which would bring to mind the likes of Greenrock's fighting pits and Freedom's gear bazaars, by comparison Gagarin appears much more like any old, slightly run-down port a captain would find around the South, only at a much grander scale. Being the center of the planet's extensive transportation system, it seems as if you are wading through thousands of Gagarians and offworlders in every atrium and hallway of this spaceport, as they do the same to make their connections on- and off-world.`
+			`	But as you make your way into the central hall you notice that, where you'd expect the greatest activity, everything has stopped. Even the ever-roving antigrav trolleys of cargo and supplies are backed up and motionless, and everybody's attention is directed at some sort of commotion in the very center of the spaceport.`
+			choice
+				`	(Take a look.)`
+				`	(Go back to your ship.)`
+					decline
+			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. You look over the bannisters with him and see, at first, the enormous, odd empty space in the middle of the hall; then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`
```

Comment:
Not sure the semicolon really works here, and that sentence seems a bit choppy. Maybe:
```suggestion
			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. Looking over the bannisters beside him, you see first the enormous and oddly empty space in the middle of the hall, then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`
```

Also, I think "bannister" is a British English spelling, and "guerilla" is an uncommon spelling, so maybe:
```suggestion
			`	Stepping through the gaps within the established crowd you end up next to a spaceport gendarme, who appears as if he's meant to be on patrol duty but is too busy looking at what everybody else is looking at to do his job at the moment. Looking over the banisters beside him, you see first the enormous and oddly empty space in the middle of the hall, then a team of overalls-wearing men armed with drills and brushes. Surrounding them is a cordon of guerrillas in their rosy CCOR uniforms, sizing up the public and the spaceport security around them with their rifles and pistols.`
```